README.md

```diff
@@ -6,17 +6,43 @@
 
 Collaborative editing is a dance.
 
-## Step one - foxychat
-
 Before biting off the actual Operation Transform implementation we have
 decided to implement a very basic chat server and web client as it holds
 a lot of the periphery tooling we will need for the ultimate goal:
 Websockets, sqlite, JWT - for details see
 [notes](https://docs.google.com/document/d/1p96DCIMo_0SB8OEVuh7U3jERzabJ1r-cJJLV0tLuR1U/edit#)
 on google docs.
 
+## Backend
+
+The foxtrot backend server is written in Go with a REST inspired HTTP
+API for chat message history and access to other resources. Websockets
+are used for new messages.
+
+### Development
+
+- Pre-requisites: [go 1.16](https://golang.org), [golangci-lint](https://github.com/golangci/golangci-lint/releases/tag/v1.33.2), GNU make
+- Build with `make`
+- View build options with `make help`
+
+### Run foxtrot
+
+Start foxtrot server with transient in-memory DB and some test sample
```

Comment:
Some missing articles: "Start *the* foxtrot server with *a* transient in-memory DB..."